packages/jest-runner/src/index.ts

```diff
@@ -90,18 +98,55 @@ class TestRunner {
               if (watcher.isInterrupted()) {
                 throw new CancelRun();
               }
+              let sendMessageToJest: JestTestFileEvent;
+
+              // Remove `if(onStart)` in Jest 27
```

Comment:
@sauravhiremath @kunal-kushwaha you wouldn't by any chance be up for sending a PR removing `onStart` etc? 🙂

Reply:
@SimenB I'll be happy to look into it 😄